pkg/analyzer_plugin/pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
   sdk: '>=2.0.0 <3.0.0'
 
 dependencies:
-  analyzer: '>=0.35.3 <0.38.0'
+  analyzer: '>=0.35.3 <0.38.4'
```

Comment:
`<0.39.0`?

Reply:
Oh! I get it now. This should be fine...I think...

Reply:
@ReginFell, if this looks good, can you make this change.

@bwilkerson - do you mind shepherding https://dart-review.googlesource.com/c/sdk/+/118721 in?

Reply:
No, I don't mind.

Reply:
Yeah, done

Reply:
Thanks!